io.github.swanux.hbud.yml

```
@@ -0,0 +1,139 @@
+app-id: io.github.swanux.hbud
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: io.github.swanux.hbud
+
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --share=ipc
+  - --device=all
+  - --filesystem=xdg-run/gvfs
+
+cleanup:
+  - "/lib/cmake"
+  - "*.la"
+  - "*.a"
+  - "/share/doc"
+  - "/share/gtk-doc"
+  - "/share/locale"
+  - "/share/runtime"
+  - "/share/kde4"
+  - "/share/metainfo"
+  - "/share/vlc"
+  - "/share/icons/hicolor/16x16"
+  - "/share/icons/hicolor/32x32"
+  - "/share/icons/hicolor/48x48"
+  - "/share/icons/hicolor/256x256"
+  - "/share/icons/hicolor/128x128/apps/vlc*.png"
+  - "/share/applications/vlc.desktop"
+  - "/bin/pylupdate5"
+  - "/bin/pyrcc5"
+  - "/bin/pyuic5"
+  - "/include"
+  - "/lib/debug"
+  - "/lib/pkgconfig"
+  - "/share/man"
+
+modules:
+  - name: io.github.swanux.hbud
+    buildsystem: simple
+    sources:
+#       - type: dir
+#         path: ../
+      - type: archive
+        url: https://github.com/swanux/hbud/archive/refs/tags/v0.3.5.tar.gz
+        sha256: ac5f5e0ba1a9f3023a00dd827951ba8dcc70d91788a59977b449bd8d27918876
+    build-commands:
+      - mkdir -p /app/share/applications
+      - mkdir -p /app/share/hbud
+      - mkdir -p /app/bin
+      - mkdir -p /app/share/appdata
+      - mkdir -p /app/share/icons/hicolor/128x128/apps
+      - cp -r DEV_FILES/hbud /app/share/hbud/.
+      - cp -r DEV_FILES/modules /app/share/hbud/. 
+      - cp DEV_FILES/io.github.swanux.hbud.desktop /app/share/applications/.
+      - cp DEV_FILES/io.github.swanux.hbud /app/share/hbud/.
+      - ln -s /app/share/hbud/io.github.swanux.hbud /app/bin/.
+      - cp DEV_FILES/io.github.swanux.hbud.appdata.xml /app/share/appdata/.
+      - cp DEV_FILES/hbud/icons/io.github.swanux.hbud.svg /app/share/icons/hicolor/128x128/apps/.
```

```suggestion
      - install -Dm644 -t /app/share/icons/hicolor/scalable/apps/ DEV_FILES/hbud/icons/io.github.swanux.hbud.svg 
```

svg are scalable.

also at that point it's better to use `install`.